cli/cmd/list/info/info.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	routetypes "github.com/agntcy/dir/api/routing/v1alpha1"
 	"github.com/agntcy/dir/cli/presenter"
-	"github.com/agntcy/dir/cli/util"
+	ctxUtils "github.com/agntcy/dir/cli/util/context"
```

Comment:
```suggestion
	util "github.com/agntcy/dir/cli/util/context"
```
this `util` -> `ctxUtils` change has not been very consistent across the PR, it would be good to either change to `ctxUtils` or to `util` not both